thrift/thrift-gen/main.go

```diff
@@ -41,14 +41,30 @@ import (
 
 const tchannelThriftImport = "github.com/uber/tchannel-go/thrift"
 
+type stringslice []string
```

Comment:
nit: camcel case, and put `flag` in the name so we know what it's for, `stringSliceFlag`